google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/StorageSnippets.java

```diff
@@ -1577,4 +1582,140 @@ public URL generateV4GPutbjectSignedUrl(String bucketName, String objectName)
     // [END storage_generate_upload_signed_url_v4]
     return url;
   }
+
+  /** Example of creating an HMAC key for a service account * */
+  public HmacKey createHmacKey(String serviceAccountEmail) throws StorageException {
+    // [START storage_create_hmac_key]
```

Comment:
I'm guessing this is part of some doc set somewhere? (A filed issue explaining all this would be helpful.) Is there documentation for how the example inclusion  works?